### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by igor0806**
- [ ] Mentor is added to pr
- [ ]  AC copied to pr description
- [ ]  Link to deployed version is added to pr
- [ ]  Simple Pagination component (without ...) is added and works correctly
- [ ]  On the 1st page button Back should be disabled, same as button Forward on the last page
- [ ]  Interaction with pagination should not change or break sorting and filtration.
- [ ]  Category filter (multi-select) is added and works correctly
- [ ]  Sorting input (single select) is added and works correctly
- [ ] 
- [ ] Interaction with filter and sorter should reset pagination and move a user to the 1st page

- [ ] Pagination component with ... is implemented according to [design]

- [ ]  Search input works correctly

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
As @MZabolotnev is on vacation, @serhii-tsivinskyi-moc  please review

File: src/components/Filters/Filters.component.tsx [hunk 1]
```diff
@@ -1,53 +1,85 @@
...
+                                <button
+                                    key={category}
+                                    className={`${styles.filterButton} ${selectedCategories.includes(category) ? styles.activeFilterButton : ''}`}
+                                    onClick={() => handleCategorySelect(category)}
+                                >
+                                    {category}
+                                </button>
+                            ))}
+                        </div>
+                        <div className={styles.sortFlex}>
```
Comment (serhii-tsivinskyi-moc) [inline src/components/Filters/Filters.component.tsx]: Screenshot 2024-06-18 at 12 14 13
filters don't matched to desing

File: src/Models/ProductList.tsx [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
```
Comment (serhii-tsivinskyi-moc) [inline src/Models/ProductList.tsx]: `ProductsList` component should be inside components folder. Use `Models` for interfaces/types

File: src/Models/ProductList.tsx [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
 
+import FiltersComponent from '@/components/Filters/Filters.component';
```
Comment (serhii-tsivinskyi-moc) [inline src/Models/ProductList.tsx]: pagination component should highlight current page

File: src/Models/ProductList.tsx [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
 import Pagination from '@/components/Pagination/Pagination.component';
+import type { CategoryEnum } from '@/pages/products/useFilters';
```
Comment (serhii-tsivinskyi-moc) [inline src/Models/ProductList.tsx]: Screenshot 2024-06-18 at 12 20 08

filters UI is broken

File: src/Models/ProductList.tsx [hunk 5]
```diff
@@ -1,7 +1,10 @@
...
+import { SortEnum } from '@/pages/products/useFilters';
```
Comment (serhii-tsivinskyi-moc) [inline src/Models/ProductList.tsx]: don't show pagination if no products found

File: src/components/Product/ProductList.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+import FiltersComponent from '@/components/Filters/Filters.component.tsx';
+import Pagination from '@/components/Pagination/Pagination.component.tsx';
+import styles from '@/components/Product/Product.module.css';
+import ProductCard from '@/components/Product/ProductCard.tsx';
+import type { Category } from '@/constants/filtersCategory.ts';
+import { SortOption } from '@/constants/filtersSortOption.ts';
+import mockData from '@/Models/mocData.json';
+import type { Product } from '@/Models/product.ts';
+
+const ProductList: React.FC = () => {
```
Comment (serhii-tsivinskyi-moc) [inline src/components/Product/ProductList.tsx:14]: Screenshot 2024-06-23 at 12 44 49

Products should be placed near each other

**Review by serhii-tsivinskyi-moc — APPROVED** [review]

**Pull request merged**